code/modules/projectiles/projectile/bullets.dm

```diff
@@ -8,6 +8,10 @@
 	sharp = 1
 	hitsound_wall = "ricochet"
 
+/obj/item/projectile/bullet/strongbullet //ouch
+	damage = 80
+	stamina = 60

```

Comment:
This is pretty much a one-shot, one-kill gun, at this stage.

It does 80 damage and instantly weakens them.